src/vscode-tree-item/vscode-tree-item.ts

```diff
@@ -142,6 +173,31 @@ export class VscodeTreeItem extends VscElement {
     this._mainSlotChange();
     this.role = 'treeitem';
     this.ariaDisabled = 'false';
+    this.addEventListener('pointerenter', this._handlePointerEnter);
```

Comment:
Component's event listeners should be setup in the constructor. I aim to follow this convention in new code. See this article: https://hawkticehurst.com/2023/11/you-are-probably-using-connectedcallback-wrong/